### PR TITLE
[ops] Small dashboard/graph fixes

### PR DESCRIPTION
**Pull request by geropl**
## Release Notes

```release-note
None
```

### CODE REVIEW

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **geropl** after the PR has been reviewed.

*No associated issue*. Update pull-request body to add a reference to an issue, or get approval with `/approve no-issue`

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[operations/observability/mixins/meta/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/operations/observability/mixins/meta/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: operations/observability/mixins/meta/dashboards/components/server.json [hunk 1]
```diff
@@ -678,7 +678,7 @@
       "targets": [
         {
           "exemplar": true,
-          "expr": "sum(server_websocket_connection_count) by (cluster, clientType)",
+          "expr": "sum(server_websocket_connection_count{cluster=~\"$cluster\", pod=~\"$pod\"}) by (cluster, clientType)",
```
Comment (laushinka) [inline operations/observability/mixins/meta/dashboards/components/server.json:681]: Would it work if we don't add `pod` to the dashboard settings variable?

Comment (geropl) [reply operations/observability/mixins/meta/dashboards/components/server.json:681]: You mean if we removed the variable `pod` from the dashboard entirely? Yes.

But this is the "Component / server" dashboard where I'd like to offer as much options for drill-down as possible. It was just that this graph did not respect those filters until now.

Comment (laushinka) [reply operations/observability/mixins/meta/dashboards/components/server.json:681]: Sorry I missed that it's in the server file! Looks good then.

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: dd2f74406efbede64d2964db396ee659d7911eef</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/6998#issuecomment-983798559" title="LGTM">laushinka</a>*

*No associated issue*. Update pull-request body to add a reference to an issue, or get approval with `/approve no-issue`

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[operations/observability/mixins/meta/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/operations/observability/mixins/meta/OWNERS)~~ [laushinka]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/6998#issuecomment-983799453" title="Approved">geropl</a>*, *<a href="https://github.com/gitpod-io/gitpod/pull/6998#issuecomment-983798559" title="LGTM">laushinka</a>*

Associated issue requirement bypassed by: *<a href="https://github.com/gitpod-io/gitpod/pull/6998#issuecomment-983799453" title="Approved">geropl</a>*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[operations/observability/mixins/meta/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/operations/observability/mixins/meta/OWNERS)~~ [geropl,laushinka]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by geropl** [comment]
/werft run

last build failed bc disk was full :roll_eyes:

**Pull request merged**